buildSrc/src/main/kotlin/pklAllProjects.gradle.kts

```diff
@@ -132,7 +132,11 @@ private fun KotlinGradleExtension.configureFormatter() {
   licenseHeaderFile(licenseHeaderFile, "([a-zA-Z]|@file|//)")
 }
 
+val originalRemoteName = System.getenv("PKL_ORIGINAL_REMOTE_NAME") ?: "origin"
+
 spotless {
+  ratchetFrom = "$originalRemoteName/main"
```

Comment:
I'd love to just say "the remote whose URL is git@github.com:apple/pkl.git", but it doesn't look like Spotless supports this.

Reply:
Yeah, true!